pkg/controller/ec2securitygroup/ec2securitygroup_controller.go

```diff
@@ -226,5 +258,16 @@ func (r *ReconcileEC2SecurityGroup) Reconcile(request reconcile.Request) (reconc
 		r.events.Event(instance, `Normal`, `Deleted`, "Deleted EC2SecurityGroup and removed finalizers")
 	}
 
+	// START EC2 KEYPAIR
```

Comment:
What are these comments all about?

Reply:
Was an outline meant for something else that I put in there by mistake. Removed.